### ISSUE TITLE
Pulsemeeter not starting after Python Update

### ISSUE THREAD

**Issue by mikeblack999** [theRealCarneiro/pulsemeeter#131]
**Describe the bug**
After the Python Update yesterday my pulsemeeter is not starting anymore
Log from console:

```
Traceback (most recent call last):
  File "/usr/lib/python3.13/importlib/metadata/__init__.py", line 407, in from_name
    return next(iter(cls.discover(name=name)))
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/pulsemeeter", line 33, in <module>
    sys.exit(load_entry_point('pulsemeeter==1.2.14', 'console_scripts', 'pulsemeeter')())
             ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/pulsemeeter", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
                       ~~~~~~~~~~~~^^^^^^^^^^^
  File "/usr/lib/python3.13/importlib/metadata/__init__.py", line 960, in distribution
    return Distribution.from_name(distribution_name)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/importlib/metadata/__init__.py", line 409, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: No package metadata was found for pulsemeeter
```

**To Reproduce**
1. Open Terminal
2. Start it with command pulsemeeter

**Expected behavior**
Normal starting as before

**Desktop (please complete the following information):**
 - Distro: EndeavourOS (Arch)
 - Python Version 3.13.1
 - Pulsemeeter installed via Arch AUR

**Comment by erdnuesse** [comment]
IIRC Python 2.x was a hard requirement for Pulsemeeter, so no Issue, since you use 3.x. 
Am I missing something? (but I'm just a user as well, so...)

**Comment by mikeblack999** [comment]
> IIRC Python 2.x was a hard requirement for Pulsemeeter, so no Issue, since you use 3.x. Am I missing something? (but I'm just a user as well, so...)

Hm I don't know, so far I didn't have a problem, I'm using Pulsemeeter for a year now and the Python update yesterday broke it and also zapzap by the way, which also requires python, so I also don't know if python needs to fix this or the apps need to be changed to be compatible with the new python libs.

**Comment by Fl1tzi** [comment]
Hi. There has been no recent updates (in like 2 years). I don't expect anyone fixing it any time soon.

**Comment by mikeblack999** [comment]
Hey all, sorry for making trouble, I just removed pulsemeeter and reinstalled it from AUR, it now works!
It's time for christmas, I can't think anymore :D

**Comment by erdnuesse** [comment]
@mikeblack999 irrelevant if it worked or not. If Python 2 was ever only supported, your version mismatch - although working for a bit - is generally a you-problem - not any dev's problem neither PM, nor Python. The latter will not fix anything, since the PM was being built against an 2.x (again, IIRC), which is EOL for about 5 Years now, so... take your pick. You shouldn't have upgraded, or better yet, you maybe should have kept an old version, when you're running EOL or unsupported code. The fact that it worked fine for you is better than most newer users could report, and maybe you can just revert your changes to the last working state. #MrHindsight

@Fl1tzi Just because you don't see anything, does not mean nothing is being done. I totally understand that it looks like this, but @theRealCarneiro now said on multiple occasions that it needs a full rework, and from what I understand about SWdev is, that in that case, tracking those changes here would be useless, since we're talking about a completely different app at this point.
Also, maybe it's just me, but pushing another App in the issues might be considered bad form, but that's not for me to decide.

**Comment by erdnuesse** [comment]
> @erdnuesse Thats why we literally started this project. I also contributed to this project and at one point just gave up. 

I was under the impression @theRealCarneiro would develop outside any publicly visible repo. 

> @theRealCarneiro has done awesome work here but personally I believe this project will either need a full rewrite (which seems like no one has time or interest in) 
As I already said above, IIRC @theRealCarneiro said they started the rewrite already, but years ago frankly.
> Sonusmix is not a rewrite of this project but transfers the idea of this project.

No one disputed, or mentioned something akin to that. 

However, I have a few issues here, @Fl1tzi  
What is good: 
- There exists a spiritual successor
- which seems to run without major problems
- you already contributed

But bad though:
- it hasn't seen a release in 2 years either
- it runs on a privately managed flatpak repo (security is an issue, even if you could be trusted as devs here)
  - meaning: I trust flathub just a tiny bit more to have a hardened server, but that's pure speculation and could just as well be the opposite, however, there's probably more eyes on potential issues. 
- optimally would be an endorsement from @theRealCarneiro that he has at least no issues when anyone starts shilling alternatives.

I already took a look at sonusmix, and while the current state seems okay, here's what I would love to see:
- a push to flathub
- better commit messages. I.e. removing the "--user" flag in the same commit where documentation was added; and the commit said documentation only. And why the app now should run in the system context by default has NO mention whatsoever. 

-> come on, we linux users are known to be picky, and you just can't slide a different context by users, when you want to be trusted. Be better, thanks for consideration.

**Comment by Fl1tzi** [comment]
> it hasn't seen a release in 2 years either

The last changes are 2 months ago. So very recent.

> it runs on a privately managed flatpak repo (security is an issue, even if you could be trusted as devs here) 

Understandable. Our point is that we wanted to test flatpaks but we want to publish to flathub once we figured out that everything is stable. Everything is build on the Codeberg CI and not on our personal machines. But ofc there is at least some risk to it.

> optimally would be an endorsement from @theRealCarneiro that he has at least no issues when anyone starts shilling alternatives.

I understand. I also talked to him about a rewrite before we stopped talking. I believe thats up to him to decide.

If you have further issues, you might want to directly publish them on the Sonusmix issue tracker cause this is off topic.
